ReactiveFeedback/Feedback.swift

```diff
@@ -40,20 +56,27 @@ public struct Feedback<State, Event> {
         }
     }
 
+    /// Creates a Feedback which will perform effects on for certain state filtered by the predicate.
+    /// Each new effect cancels a previous one
+    /// - parameters:
+    ///    - predicate: A closure which defines weather effect should be performed to particular value of the `State`
```

Comment:
`A closure which defines whether an effect should be performed to a particular value of the `State``